definitions/types/gcp-infrastructure-bucket.json

```
@@ -0,0 +1,28 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "type": "object",
+  "title": "GCP Bucket",
+  "description": "Google Cloud Storage Bucket",
+  "additionalProperties": false,
+  "required": [
+    "id"
+  ],
+  "properties": {
+    "id": {
+      "type": "string",
+      "title": "GCS Bucket ID",
+      "description": "The Globally Unique ID of the GCS Bucket",
+      "examples": [
+        "one",
+        "dash-bucket",
+        "snake_bucket",
+        "domain.type.bucket",
+        "1also-valid2"
+      ],
+      "pattern": "^[a-z0-9]+[a-z0-9-_.]{1,61}[a-z0-9]+$",
```

```suggestion
      "pattern": "^[a-z0-9][a-z0-9-_.]{1,61}[a-z0-9]$",
```

The `{1,61}` length requirement here is effectively ignored since the first and last `[a-z0-9]` have the `+` modifier (1 or more).  If you drop the `+` you get a hard requirement of 3-63 characters that starts and ends w/ a lowercase alphanumeric.